src/v2/controllers/FormController.js

```diff
@@ -109,6 +109,10 @@ export default Controller.extend({
   },
 
   handleInvokeAction (actionPath = '') {
+    if(actionPath === 'cancel') {
```

Comment:
Curious what's the response of `cancel`. Any examples?

Reply:
Any sign out after entering username is a cancel.. Any other actionPath could be `cancel` ?
![Screen Shot 2021-01-12 at 10 25 54 AM](https://user-images.githubusercontent.com/38230587/104356308-a5f9d600-54c0-11eb-9003-677aa686852b.png)